wp-includes/class-wp-web-app-manifest.php

```diff
@@ -119,7 +119,7 @@ public function get_theme_color() {
 	public function get_manifest() {
 		$manifest = array(
 			'name'      => wp_kses_decode_entities( get_bloginfo( 'name' ) ),
-			'start_url' => get_home_url(),
+			'start_url' => user_trailingslashit( get_home_url() ),
```

Comment:
Shouldn't this be `trailingslashit()`?

Reply:
I have tested it with and without a trailing slash in the home url and as I understand user_trailingslashit documentation it will check if the permalink settings has a trailing slash.

"Conditionally adds a trailing slash if the permalink structure has a trailing slash, strips the trailing slash if not."

https://codex.wordpress.org/Function_Reference/user_trailingslashit

Reply:
What about instead just doing. 

```php
'start_url' => home_url( '/' ),
```

This is what core does to get the URL to the front page: https://github.com/WordPress/wordpress-develop/blob/a8a4c09f33356475f4a3f3a9f5bdacd0c25bcb42/src/wp-includes/link-template.php#L332-L333